pkg/kubelet/cm/cgroup_manager_linux.go

```diff
@@ -246,7 +246,7 @@ func updateSystemdCgroupInfo(cgroupConfig *libcontainerconfigs.Cgroup, cgroupNam
 }
 
 // Exists checks if all subsystem cgroups already exist
-func (m *cgroupManagerImpl) Exists(name CgroupName) bool {
+func (m *cgroupManagerImpl) Exists(name CgroupName) error {
```

Comment:
Changing a function like `Exists` to return a non-bool seems a bit odd to me. Because the question being asked here is "does it exist?", not "is there an error in my CgroupName?". Consider renaming the method to `Validate` or similar to indicate that what the method is doing is actually validating that the CgroupName is valid for the cgroup manager.

Reply:
+1 I think this is a great point @jaypipes 

Another option is that we could keep the `Exists` call as is, and instead have the functions calling `Exists` be responsible for logging errors when appropriate.